api/descriptions/feeds/custom_vulnerabilities_get.md

```diff
@@ -1,17 +1,41 @@
-Retrieves the list of vulnerabilities for internally developed packages.
+Retrieves the list of custom vulnerabilities and associated rules for handling internally created or packaged apps.
 
-These entries are used by the Twistlock scanner to detect vulnerable custom components (apps, libraries, etc) that were developed and packaged internally.
+This list is used by the Twistlock scanner to detect vulnerable custom components (apps, libraries, etc) that were developed and packaged internally.
 
-**NOTE**: When a vulnerable custom component is detected in an image, you must have a rule to tell Twistlock how to handle it.
-Vulnerability rules can be created in the Console UI or with the /api/v1/policies/cve endpoint.
-Create a vulnerability rule and set ID 412 (Image contains vulnerable custom components) to ignore, alert, or block.
+**Note:** When a vulnerable custom component is detected in an image, you must have a rule to tell Twistlock how to handle it. Vulnerability rules can be created using the Console UI or with the /api/v1/policies/cve endpoint. To set a vulnerability rule to ignore, alert, or block, use the rule ID `412` to indicate the image contains vulnerable custom components. See `/api/v1/policies/cve` endpoint for more info.
```

Comment:
@KICKdesigns Some of this is no longer true:

* The `/api/v1/policies/cve` endpoint no longer exists. It's been replaced with `/api/v1/vulnerabilty/<RESOURCE-TYPE>` endpoints.
* More importantly, no specific rule needs to be created for us to report when we find custom vulnerable components. As long as you have at least one vuln rule (and you will, because we have a default `Default - alert all components` rule for all resource types), we report any findings. As part of this, there is no `412` rule ID.